.github/workflows/publish_on_tag.yaml

```diff
@@ -0,0 +1,41 @@
+name: Build and Push Custom Component
```

Comment:
🔥 Let's also add a testing step in the CI + documentation (e.g. how to trigger, where to put API key etc.)

Reply:
let's do it in a separate PR